frontend/src/utils/auth.js

```diff
@@ -0,0 +1,18 @@
+const localStorageKey = "til_token";
+
+// TOKENS
+export const deleteToken = () => {
+  window.localStorage.removeItem(localStorageKey);
+};
+
+export const getToken = () => {
+  return window.localStorage.getItem(localStorageKey);
+};
+
+// AUTHENTICATION
+export const isAuthenticated = () => {
+  const token = getToken();
+  if (token) return true;
```

Comment:
add empty line before if statement
```suggestion
  if (token) {
    return true;
  }
```